internal/handlers/common/update.go

```diff
@@ -118,5 +122,77 @@ func UpdateDocument(doc, update *types.Document) (bool, error) {
 		}
 	}
 
-	return true, nil
+	return changed, nil
+}
+
+// ValidateUpdateOperators validates update statement.
+func ValidateUpdateOperators(update *types.Document) (err error) {
```

Comment:
Let's not use a named result if we don't use it

Reply:
fixed